src/client/containers/OrderConfirmationPage/OrderConfirmationPage.Container.js

```diff
@@ -2,5 +2,30 @@ import React from 'react';
 import './OrderConfirmationPage.styles.css';
 
 export default function OrderConfirmationPage() {
-  return <div>Order Confirmation Page</div>;
+  return <div className="confirmation-page">
+    <div className="inner-div">
+      <div className="wrapper">
+        <div className="wrapper-content">
+          <h3>
+            THANK YOU FOR YOUR ORDER
+          </h3>
+          <p>
+            We have sent a receipt to your email
+          </p>
+        </div>
+      </div>
+      <div className="wrapper2">
+      <div className="wrapper-content2">
```

Comment:
I would recommend extracting event handlers to named functions instead, it makes it much easier to format and read the markup.

```js
const continueShopping = () => {
  window.location.href = '/';
}

<button type='button' onClick={continueShopping} className="continue-shopping-button">CONTINUE SHOPPING</button>
```